lib/registry/src/lib.rs

```diff
@@ -113,8 +115,9 @@ pub fn get_executable_file_from_path(
     package_dir: &PathBuf,
     command: Option<&str>,
 ) -> Result<(wapm_toml::Manifest, PathBuf), anyhow::Error> {
-    let wapm_toml = std::fs::read_to_string(package_dir.join("wapm.toml"))
-        .map_err(|_| anyhow::anyhow!("Package {package_dir:?} has no wapm.toml"))?;
+    let wapm_toml = std::fs::read_to_string(package_dir.join(GLOBAL_CONFIG_FILE_NAME))
```

Comment:
Let's name this wasmer_toml.

Reply:
This is for putting it in the package. You told me to use wasmer.toml for uploading, but put wapm.toml in the .tar.gz (to be compatible with the current model). Not sure if this will break downloading / running packages.

Reply:
@syrusakbary this will break wasmer run, because all packages on wapm use `wapm.toml`. I can either break it, or prefer wasmer.toml and look for wapm.toml as a fallback.

Reply:
I meant the naming of the variable `wapm_toml` seems incorrect. We should name it `wasmer_toml`

> prefer wasmer.toml and look for wapm.toml as a fallback.

In any case, this seems like the right approach for the file name

Reply:
Okay, done, I've also fixed `std::fs::read("wapm.toml")` to use a new function to prefer wasmer.toml and use wapm.toml as a fallback.